nucypher/characters/lawful.py

```diff
@@ -1735,6 +1738,62 @@ def _reencrypt(self, kfrag: KFrag, work_order: 'WorkOrder', alice_verifying_key:
         # ... and finally returns all the re-encrypted bytes
         return cfrag_byte_stream
 
+    def status_info(self, omit_known_nodes=False) -> 'LocalUrsulaStatus':
+
+        domain = self.domain
+        version = nucypher.__version__
+
+        latest_states = self.known_nodes.latest_states(5)
+
+        fleet_state = latest_states[-1]
```

Comment:
What is the expected behavior here if `latest_states` is empty?

Reply:
It is never empty, one state is created on `FleetSensor` creation. Although, admittedly, it is not obvious from this context.

Reply:
To make it more obvious from the API there could be a `latest_state()` (returning a single state) and `previous_states()` (returning `states[...:-1]`) methods. That's how the info is used in building the status anyway.

Reply:
That sounds great.  An elegant and simple way to improve readability here.

Reply:
Fixed.